subprojects/enterprise/src/main/java/org/gradle/internal/enterprise/GradleEnterprisePluginBuildState.java

```diff
@@ -40,4 +41,5 @@
     @Nullable // if not a daemon build
     DaemonScanInfo getDaemonScanInfo();
 
+    StartParameter getStartParameter();
```

Comment:
I think it would be good to not expose `StartParameter` on the new stable API, so we can gradually get rid of it - at least internally, and perhaps instead reuse the mechanism we added for observing the configuration cache and file watching flags.

Reply:
I was assuming that when we could eventually get rid of it, we would just stub this out. Moving to ops for everything _now_ is not feasible. If you prefer, I can wrap this in a cut down, specific, interface.

Reply:
Let's leave it like this, then. I wouldn't bother wrapping at this stage. `StartParameter` is already part of the stable API and we can gradually move to ops and remove it from the stable API.